tests.py

```diff
@@ -311,6 +311,28 @@ def test_import_from_qtwidgets():
     assert QPushButton.__name__ == "QPushButton", QPushButton
 
 
+def test_pyside2uic():
```

Comment:
It would be allright, but [the addition you made](https://github.com/mottosso/Qt.py/pull/142/files#diff-d5856ea6ea00e434586174344f638824R22), is it still uncertain whether or not that works?

We should find that out first, so we know whether we can rely on that the errors it gives us is related to Qt.py, and not the way we installed it.